tests/benchdnn/CMakeLists.txt

```diff
@@ -73,7 +73,10 @@ function(register_benchdnn_test engine driver test_file)
     endif()
 
     set(mode_modifier "")
-    if(${engine} MATCHES "gpu" AND NOT WIN32)
+    if(${engine} MATCHES "gpu"
```

Comment:
Could you please add a comment on why it's removed for posterity?

Reply:
Added a comment with brief explanation of change.

Reply:
Thanks, @ShanoToni!